dissect/target/loaders/velociraptor.py

```diff
@@ -87,11 +87,13 @@ def __init__(self, path: Path, **kwargs):
         super().__init__(path)
 
         if path.suffix == ".zip":
-            log.warning(
-                f"Velociraptor target {path!r} is compressed, which will slightly affect performance. "
-                "Consider uncompressing the archive and passing the uncompressed folder to Dissect."
-            )
             self.root = zipfile.Path(path.open("rb"))
+            compression_type = zipfile.ZipFile(str(path)).getinfo("uploads.json").compress_type
```

Comment:
```suggestion
            compression_type = self.root.root.getinfo("uploads.json").compress_type
```
I didn't test this but I think this achieves the same without reopening the archive twice.

Reply:
Fixed